src/lib/adapters/slack-like.js

```diff
@@ -56,6 +56,11 @@ SlackLikeAdapter.prototype.normalizeCommand = function(command) {
   command = command.replace(/\u2018/g, '\u0027');
   // replace right single quote with regular apostrophe
   command = command.replace(/\u2019/g, '\u0027');
+  // replace non-breaking space with regular space
+  // first the unicode variant, so we don't cut it in half,
+  // then the latin1 variant (thanks to slack)
+  command = command.replace(/\u00A0/g, ' ');
+  command = command.replace(/\xA0/g, ' ');
```

Comment:
NAK.

The `SlackLikeAdapter` is used by any adapter that uses a Slack-like interface, but not all of them may have this issue. Since you are only adding tests for the Slack adapter and not all of the SlackLikeAdapters, this would probably only apply to the Slack adapter itself. That means that the better place for this would be in `src/lib/adapters/slack.js`.

Within the SlackAdapter, I would do it this way:

```js
SlackAdapter.prototype.normalizeCommand = function (command) {
  SlackLikeAdapter.prototype.normalizeCommand.call(this, command);
  // replace non-breaking space with regular space
  // first the unicode variant, so we don't cut it in half,
  // then the latin1 variant (thanks to slack)
  command = command.replace(/\u00A0/g, ' ');
  command = command.replace(/\xA0/g, ' ');
  return command;
}
```

I am writing this code off the cuff, so definitely test it out.

Reply:
done. Plus added version bump and changelog entry